3-reveal/migrations/5-IRS/2-Zambia-2019/deploy/create_structure_geo_hierarchy.psql

```diff
@@ -22,16 +22,16 @@ BEGIN
         jurisdictions.geographic_level AS jurisdiction_depth,
         geo_jurisdictions.id AS geo_jurisdiction_id,
         geo_jurisdictions.geographic_level AS geo_jurisdiction_depth,
-        ST_WITHIN(locations.geometry, geo_jurisdictions.geometry) AS geo_strict_within
-    INTO STRICT geo_record
+        public.ST_WITHIN(locations.geometry, geo_jurisdictions.geometry) AS geo_strict_within
+    INTO geo_record
     FROM locations
     LEFT JOIN jurisdictions
         AS geo_jurisdictions
-        ON ST_DWITHIN(locations.geometry, geo_jurisdictions.geometry, 150.0 / 111111.0)
+        ON public.ST_DWITHIN(locations.geometry, geo_jurisdictions.geometry, 150.0 / 111111.0)
```

Comment:
can we set use_spheroid the final arg to false? docs say this is faster

https://postgis.net/docs/ST_DWithin.html

Reply:
this isn't relevant nevermind

Reply:
I tried playing around with this, but it [breaks the tests](https://app.circleci.com/pipelines/github/OpenSRP/opensrp-reveal-datawarehouse/53/workflows/e5426b4e-58bc-422c-a896-812f36666f96/jobs/54).  I have created an issue to investigate this further later: https://github.com/OpenSRP/opensrp-reveal-datawarehouse/issues/16

Reply:
oh yea I added a comment on the other line explaining that I was wrong to suggest this, just copied to this issue, https://github.com/OpenSRP/opensrp-reveal-datawarehouse/issues/16#issuecomment-682139533